WordPress/Classes/ViewRelated/Comments/EditReplyViewController.m

```diff
@@ -11,6 +40,122 @@ - (void)viewDidLoad
+                                                              relatedBy:NSLayoutRelationEqual
+                                                                 toItem:self.textView
+                                                              attribute:NSLayoutAttributeLeft
+                                                             multiplier:1
+                                                               constant:0]];
+        
+        [self.view addConstraint:[NSLayoutConstraint constraintWithItem:self.suggestionsTableView
+                                                              attribute:NSLayoutAttributeRight
+                                                              relatedBy:NSLayoutRelationEqual
+                                                                 toItem:self.textView
+                                                              attribute:NSLayoutAttributeRight
+                                                             multiplier:1
+                                                               constant:0]];
+        
+        // Pin the suggestions view top to the super view top, less a 30 pt margin
+        [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|-30.0-[suggestionsview]"
+                                                                          options:0
+                                                                          metrics:nil
+                                                                            views:views]];
+        
+        // Pin the suggestions view bottom to the bottom of the reply box
+        [self.view addConstraint:[NSLayoutConstraint constraintWithItem:self.suggestionsTableView
+                                                              attribute:NSLayoutAttributeBottom
+                                                              relatedBy:NSLayoutRelationEqual
+                                                                 toItem:self.textView
+                                                              attribute:NSLayoutAttributeBottom
+                                                             multiplier:1
+                                                               constant:0]];
+        
+        // listen to textView changes
+        self.textView.delegate = self;
+    }
+}
+
+#pragma mark - UITextViewDelegate
+
+- (BOOL)textView:(UITextView *)textView shouldChangeTextInRange:(NSRange)range replacementText:(NSString *)text
+{
+    // Get all the text
+    NSString *textViewText = textView.text;
+    NSRange textViewRange = NSMakeRange(0, range.location);
+    NSString *pretext = [[textViewText substringWithRange:textViewRange] stringByAppendingString:text];
+    NSArray *words = [pretext componentsSeparatedByCharactersInSet:[NSCharacterSet whitespaceAndNewlineCharacterSet]];
+    [self.suggestionsTableView showSuggestionsForWord:[words lastObject]];
```

Comment:
(I think) all of this code is already part of the ReplyTextView's `shouldChangeTextInRange` implementation.

(Unless i'm missing something) this method could be replaced with something like this:

```
- (void)textView:(UITextView *)textView didTypeWord:(NSString*)word
{
    // Show suggestions here
}
```